aggregateur/main.py

```diff
@@ -161,18 +160,11 @@ def validator(self):
             raise NotImplementedError
 
     def remote_available(self):
-        # Check that a remote host is available on TCP 443 with a 5s timeout
-
-        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        sock.settimeout(5)
-
-        parsed = urlparse(self.git_url)
-        if parsed.scheme != "https":
-            raise NotImplementedError(
-                f"Only clone/pull over TCP 443 is supported for now"
-            )
-
-        return sock.connect_ex((parsed.hostname, 443)) == 0
+        try:
+            r = requests.head(self.git_url, allow_redirects=True, timeout=5)
```

Comment:
5 seconds might be a bit low, maybe 15 or 30? I noticed it was a bit slow (~2 seconds) on GitHub under normal circumstances.

Reply:
Went for 15s!